SQL/Archive/17.1/2016-11-21-CreateUserSitesRelTable.sql

```diff
@@ -0,0 +1,21 @@
+WARNINGS;
```

Comment:
Tells MySQL to print the warning message instead of simply indicate that there is warning that should be looked at.
Demo:
without WARNINGS;
```
mysql> INSERT INTO test_names (test_name) VALUE ('bob');
Query OK, 1 row affected (0.02 sec)
mysql> INSERT IGNORE INTO test_names (test_name) VALUE ('bob');
Query OK, 0 rows affected, 1 warning (0.04 sec)
mysql> show warnings;
+---------+------+-------------------------------------------+
| Level   | Code | Message                                   |
+---------+------+-------------------------------------------+
| Warning | 1062 | Duplicate entry 'bob' for key 'Test_name' |
+---------+------+-------------------------------------------+
```

With WARNINGS;
```
mysql> WARNINGS;
Show warnings enabled.
mysql> INSERT IGNORE INTO test_names (test_name) VALUE ('bob');
Query OK, 0 rows affected, 1 warning (0.03 sec)

Warning (Code 1062): Duplicate entry 'bob' for key 'Test_name'
```